src/CAREUI/misc/PrintPreview.tsx

```diff
@@ -12,24 +12,35 @@ type Props = {
   disabled?: boolean;
   className?: string;
   title: string;
+  onClose?: boolean;
+  closeFeature?: (value: boolean) => void;
 };
 
 export default function PrintPreview(props: Props) {
-  const normalScale = useBreakpoints({ default: 0.44, md: 1 });
+  const normalScale = useBreakpoints({ default: 1, md: 1 });
```

Comment:
@modamaan changes to this file is discouraged unless it's actually solving something. This affects all usages of the Print Preview component such as Prescriptions Print Page, etc...

The default scale set to 0.44 was intentional for previewing it in mobile.

Reply:
You should be able to achieve all the requirements to solve the issue without needing to modify the PrintPreview component ideally.

Reply:
scale updated,
If we are wrapping using the PrintPreview component, enabling the close feature is necessary to handle responsiveness